distributed/deploy/old_ssh.py

```diff
@@ -349,12 +350,27 @@ def __init__(
         nanny_port=None,
         remote_dask_worker="distributed.cli.dask_worker",
         local_directory=None,
+        **kwargs,
     ):
 
         self.scheduler_addr = scheduler_addr
         self.scheduler_port = scheduler_port
         self.nthreads = nthreads
-        self.nprocs = nprocs
+        if kwargs.get("nprocs") is not None and n_workers is not None:
+            raise ValueError(
+                "Both nprocs and n_workers were specified. Use n_workers only."
+            )
+        elif kwargs.get("nprocs") is not None:
+            warnings.warn(
+                "The nprocs argument will be removed in a future release. It has been "
+                "renamed to n_workers.",
+                FutureWarning,
+            )
+            n_workers = kwargs["nprocs"]
+        elif n_workers is None:
+            n_workers = 1
+
+        self.n_workers = n_workers
```

Comment:
Can we add a small `self.nprocs` property which redirects users to `self.n_workers` instead?

Reply:
I'm happy to add this. However, setting `nprocs` or `n_workers` as an attribute (after `__init__()` runs) doesn't actually change anything about the cluster, unless the user subsequently runs `add_worker()`. For that matter, it's not all that clear to me what the behavior should be... do all the worker nodes need to be updated with the number of worker processes? Is there a method to do that (there doesn't seem to be from my quick read of the code)?